content/ai/concepts/search-algorithms/terms/a-star-search/a-star-search.md

```diff
@@ -0,0 +1,99 @@
+---
+Title: 'A* Search'
+Description: 'An informed graph searching algorithm that efficiently determines a path between nodes based on an evaluation function.'
+Subjects:
+  - 'Data Science'
+  - 'AI'
+Tags:
+  - 'Search Algorithms'
+  - 'AI'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/machine-learning-ai-engineering-foundations'
+---
+
+**A\* Search** is an informed best-first search algorithm that efficiently determines the lowest cost path between any two nodes in a directed weighted graph with non-negative edge weights. This algorithm is a variant of Dijkstra's algorithm. A slight difference arises from the fact that an evaluation function is used to determine which node to explore next. The algorithm is correct for the same reason as Dijkstra's algorithm.
```

Comment:
Link to the best-first search entry.

Reply:
@SSwiniarski, I have made all of the changes you recommended. However, the pictures do not appear as they do not exist in the media file on the main branch. I imagine this will be fixed when the entry is merged into the main branch. Thank you.